game_eggs/steamcmd_servers/stationeers/stationeers_vanilla/egg-stationeers.json

```diff
@@ -4,50 +4,68 @@
         "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2022-12-30T11:39:51+01:00",
-    "name": "Stationeers",
-    "author": "eggs@goover.dev",
-    "description": "Stationeers Server",
-    "features": [
-        "steam_disk_space"
-    ],
+    "exported_at": "2024-02-21T04:35:19+01:00",
+    "name": "Stationeer (Vanilla)",
+    "author": "jeremiejt38@gmail.com",
+    "description": "Stationeers Server Vanilla\r\nOfficial Server Guide : https:\/\/stationeers-wiki.com\/Dedicated_Server_Guide",
+    "features": null,
     "docker_images": {
-        "ghcr.io\/parkervcp\/steamcmd:ubuntu": "ghcr.io\/parkervcp\/steamcmd:ubuntu"
+        "Wine Staging": "ghcr.io\/parkervcp\/yolks:wine_latest"
```

Comment:
Why Wine? There is a Linux server

Reply:
Old egg wasn't working, some people asking for a working egg so i decide to try with Wine and Winetricks to install VCRUN2015-17-19 who's needed for Windows Version. 
https://stationeers-wiki.com/Dedicated_Server_Guide  -> See Requirements